### PR TITLE
Hotfix/social media updates

### PR DESCRIPTION
**Pull request by VirginiaDooley**
co-authored with @Bekabyx 

This work:

- removes the twitter preview as a result of limited access to the twitter
- moves the twitter username to the contact card
- imports and stores mastodon username on the `Person` model
- renders the mastodon username (as a url for simplicity) on the contact card

To test, import people and ballots. Check a candidate in a current election with a mastodon account listed in their contact details. 

Screenshot 2023-07-12 at 11 52 41 AM

### CODE REVIEW

**Comment by VirginiaDooley** [comment]
[Remove Twitter preview/Add Mastodon profiles on WCIVF](https://trello.com/c/K27dXm47/3404-remove-twitter-preview-add-mastodon-profiles-on-wcivf)

**Comment by symroe** [comment]
Have you tested that the profile data is imported locally? Do you need to update the import from YNR process to pull this in?

**Comment by VirginiaDooley** [comment]
> Have you tested that the profile data is imported locally? Do you need to update the import from YNR process to pull this in?

Not yet, sorry - still a WIP

File: wcivf/apps/people/templates/people/includes/_person_contact_card.html [hunk 1]
```diff
@@ -26,6 +26,27 @@ <h2 class="ds-candidate-name ds-h3">
...
+                            {{ object.twitter_username}}
+                        </a>
+                    </dd>
+                </div>
+            {% endif %}
+            {% if object.mastodon_username %}
+                <div>
+                    <dt>{% trans "Mastodon" %}</dt>
+                    <dd>
+                        <a href="{{ object.mastodon_username}}" title="{% blocktrans trimmed with person_name=object.name%}{{ person_name }}'s Twitter profile{% endblocktrans%}">
```
Comment (Bekabyx) [inline wcivf/apps/people/templates/people/includes/_person_contact_card.html:43]: I think "Twitter" here needs to be changed to "Mastodon"

**Comment by Bekabyx** [comment]
Been looking into the Masto not pulling through to WCIVF issue. Got a local version of Who talking to a local version of YNR so I could tinker. Turns out YNR needs an update to one of its serialisers to give us the Masto username. I've sent a patch over via Slack so you can verify that it works 😁

**Review by Bekabyx — APPROVED** [review]

**Pull request merged**